packages/crashlytics/ios/RNFBCrashlytics/RNFBCrashlyticsInitProvider.m

```diff
@@ -21,37 +21,54 @@
 #import "RNFBPreferences.h"
 #import "RNFBJSON.h"
 #import "RNFBMeta.h"
+#import "RNFBApp/RNFBSharedUtils.h"
 
 NSString *const KEY_CRASHLYTICS_DEBUG_ENABLED = @"crashlytics_debug_enabled";
 NSString *const KEY_CRASHLYTICS_AUTO_COLLECTION_ENABLED = @"crashlytics_auto_collection_enabled";
 
 @implementation RNFBCrashlyticsInitProvider
 
 + (void)load {
-  if ([self isCrashlyticsCollectionEnabled]) {
```

Comment:
If we leave this conditional in and `isCrashlyticsCollectionEnabled` is `true` everything's fine. But if we then change `isCrashlyticsCollectionEnabled` to `false`, we never register the library. If we never register the library, we never end up calling `setCrashlyticsCollectionEnabled:false` and actually turning off the persisted setting.